ports/rp2/machine_uart.c

```diff
@@ -278,6 +320,17 @@ STATIC mp_obj_t machine_uart_make_new(const mp_obj_type_t *type, size_t n_args,
             gpio_set_outover(self->tx, GPIO_OVERRIDE_INVERT);
         }
 
+        // Set hardware flow control if configured.
+        if (self->flow != 0) {
```

Comment:
The way this is done, it won't be possible to turn flow control off if it has previously been turned on.

I think it should act more like `invert`:
- default is -1
- set `self->flow = arg[ARG_flow].u_int` if arg is not the default
- remove this `if`, ie always execute the body of this if-statement

Reply:
I see what you mean, okay fixed.